src/grading-settings/grading-sidebar/GradingSidebar.test.jsx

```diff
@@ -1,31 +1,17 @@
-import React from 'react';
-import { IntlProvider, injectIntl } from '@edx/frontend-platform/i18n';
-import { render } from '@testing-library/react';
-
+import { initializeMocks, waitFor, getByText } from '../../testUtils';
 import messages from './messages';
-import GradingSidebar from '.';
-
-const mockPathname = '/foo-bar';
-
-jest.mock('react-router-dom', () => ({
-  ...jest.requireActual('react-router-dom'),
-  useLocation: () => ({
-    pathname: mockPathname,
-  }),
-}));
-
-const RootWrapper = () => (
-  <IntlProvider locale="en" messages={{}}>
-    <GradingSidebar intl={injectIntl} courseId="123" />
-  </IntlProvider>
-);
 
 describe('<GradingSidebar />', () => {
+  beforeEach(async () => {
+    initializeMocks();
+  });
+
   it('renders sidebar text content correctly', () => {
-    const { getByText } = render(<RootWrapper />);
-    expect(getByText(messages.gradingSidebarTitle.defaultMessage)).toBeInTheDocument();
-    expect(getByText(messages.gradingSidebarAbout1.defaultMessage)).toBeInTheDocument();
-    expect(getByText(messages.gradingSidebarAbout2.defaultMessage)).toBeInTheDocument();
-    expect(getByText(messages.gradingSidebarAbout3.defaultMessage)).toBeInTheDocument();
+    waitFor(() => {
+      expect(getByText(messages.gradingSidebarTitle.defaultMessage)).toBeInTheDocument();
+      expect(getByText(messages.gradingSidebarAbout1.defaultMessage)).toBeInTheDocument();
+      expect(getByText(messages.gradingSidebarAbout2.defaultMessage)).toBeInTheDocument();
+      expect(getByText(messages.gradingSidebarAbout3.defaultMessage)).toBeInTheDocument();
+    });
```

Comment:
You still need a `render` here.

Import `import { initializeMocks, waitFor, getByText, render, screen } from '../../testUtils';` above, and then here:


```suggestion
    render(<GradingSidebar intl={injectIntl} courseId="123" />);
    expect(await screen.findByText(messages.gradingSidebarTitle.defaultMessage)).toBeInTheDocument();
    expect(screen.getByText(messages.gradingSidebarAbout1.defaultMessage)).toBeInTheDocument();
    expect(screen.getByText(messages.gradingSidebarAbout2.defaultMessage)).toBeInTheDocument();
    expect(screen.getByText(messages.gradingSidebarAbout3.defaultMessage)).toBeInTheDocument();
```

Reply:
Implemented in follow-up commit: [`7b692fc` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/7b692fc845df42de8e97aabc82a7a38e7483b5cc)